sample-bundle-scr/pom.xml

```diff
@@ -52,14 +52,16 @@
             </goals>
           </execution>
         </executions>
-        <configuration>
-          <bnd><![CDATA[
```

Comment:
Don't we need to import the packages we want to use? Are they automatically imported?

Reply:
No, That is what the bnd-maven-plugin is for. It analyses the compiled classes, looks for usage of external classes. It then writes the Imports accordingly. It will create the component xmls as well.

If you want to Export something, the best way is to add a package-info.java with the following content:
```
@org.osgi.annotation.bundle.Export
package my.package.name;
```

I'm not sure if you need to export your class, because this is usually necessary when somebody else explicitly uses your Implementation in his code.